source/patterns/@aws-solutions-constructs/aws-s3-step-function/README.md

```diff
@@ -91,8 +88,8 @@ Out of the box implementation of the Construct without any override will set the
 * Retain the S3 Bucket when deleting the CloudFormation stack
 * Applies Lifecycle rule to move noncurrent object versions to Glacier storage after 90 days
 
-### AWS CloudTrail
-* Configure a Trail in AWS CloudTrail to log API events in Amazon S3 related to the Bucket created by the Construct
+### AWS S3 Event Notification
+* Enable S3 to send events to EventBridge when an object is created.
```

Comment:
Is it just when an object is created?

Reply:
Yes, following the previous method it was only trigger for when objects are created. There are other event types such as ObjectDeleted, ObjectRestoreInitiated, ObjectRestoreCompleted, ObjectRestoredExpired, ObjectClassChanged, ObjectAccessTierChanged, ObjectACLUpdated, and ObjectTagsAdded.